Taskfile.yml

```diff
@@ -93,41 +93,58 @@ tasks:
           exit 1
         fi
 
+  start-faktory:
+    desc: Start Faktory and opslevel-runner in faktory mode
+    dir: "{{.SRC_DIR}}"
+    deps:
+      - install-goreman
+      - install-faktory
+    cmds:
+      - goreman start
 
   # internal (not directly called) tasks
 
-  go-install-tool:
-    desc: go install '{{.GO_TOOL}}' and set GOBIN if not set
+  install-changie:
+    desc: go install "changie"
     internal: true
-    silent: true
-    vars:
-      IS_TOOL_INSTALLED:
-        sh: which {{.GO_TOOL}} > /dev/null || echo "1"
+    status:
+      - test -n "command -v changie"
     cmds:
-      - test -z "{{.IS_TOOL_INSTALLED}}" || echo "Installing {{.GO_TOOL}}..."
-      - test -z "{{.IS_TOOL_INSTALLED}}" || go install {{.GO_TOOL_PATH}}
-      - test -n $(go env GOBIN) || go env -w GOBIN=$(go env GOPATH)/bin
-      - echo "  '{{.GO_TOOL}}' is installed."
-    requires:
-      vars: [GO_TOOL, GO_TOOL_PATH]
+      - go install "github.com/miniscruff/changie@latest"
 
-  install-changie:
-    desc: go install "changie"
+  install-goreman:
+    desc: go install "goreman"
+    internal: true
+    status:
+      - test -n "command -v goreman"
+    cmds:
+      - go install "github.com/mattn/goreman@latest"
+
+  install-faktory:
+    desc: install "faktory"
     internal: true
+    status:
+      - test -n "command -v faktory"
     cmds:
-      - task: go-install-tool
-        vars: { GO_TOOL: "changie", GO_TOOL_PATH: "github.com/miniscruff/changie@latest" }
+      - mkdir -p ./faktory
```

Comment:
@davidbloss I have to do this to a directory because the tarball includes a README.md that overrides ours